### PR TITLE
Exposes cancel on the wrapper

### PR DESCRIPTION
**Pull request by jakob-o**
Hi,
in some situations I just want to cancel pending executions. This should be easily done by exposing cancel on the wrapper. I complemented the tests and hope this is fine for you.
Regards
Jakob

### CODE REVIEW

File: src/debounce.js [hunk 1]
```diff
@@ -43,6 +43,11 @@ angular.module('rt.debounce', []).factory('debounce', function ($timeout) {
             }
             return result;
         };
+        
+        // Cancels the queued execution if any
+        wrapper.cancel = function () {
+            cancel();
+        };
```
Comment (rubenv) [inline src/debounce.js]: This can be simpler:

```js
wrapper.cancel = cancel;
```

**Comment by rubenv** [comment]
Makes sense to have it! Will fix the one remark above and merge it. Thanks a lot!

File: src/debounce.js (hunk 1 above)
Comment (jakob-o) [inline src/debounce.js]: Fair enough ;)

**Comment by rubenv** [comment]
Released `angular-debounce@1.0.2` with your changes, thanks a lot!

**Pull request merged**